easybuild/easyconfigs/r/R/R-4.0.0-foss-2020a.eb

```diff
@@ -114,7 +114,11 @@ exts_list = [
         'checksums': ['b6d056f75d45e315943a4618f5f62802612cd8931ba3f9f474b595140a3cfb93'],
     }),
     ('ff', '2.2-14.2', {
-        'checksums': ['f8c06ac333ffe3545cdf56531619865176e1827728f7aeeba82d135d5c5e37e4'],
+        'checksums': [
+            'f8c06ac333ffe3545cdf56531619865176e1827728f7aeeba82d135d5c5e37e4',
+            '164fe7d3aa490c9604b02bcfbdd2ea178a288362b521bb8bdac8588efaf78b8d'  # ff_aarch64_support.patch
+        ],
+        'patches': ['ff_aarch64_support.patch'],
```

Comment:
@terjekv Do you mind renaming this patch file to `ff-2.2-14.2_aarch64_support.patch`?

`ff` is very generic so adding the version helps diversify it a bit, and that makes it clear for which version the patch is intended.

Reply:
Ack. Changed name of the patch.